chain/network/src/peer_manager/network_state/tier1.rs

```diff
@@ -27,16 +39,106 @@ impl super::NetworkState {
         self: &Arc<Self>,
         clock: &time::Clock,
     ) -> Vec<Arc<SignedAccountData>> {
+        // Tier1 advertise proxies calls should be disjoint,
+        // to avoid a race condition while connecting to the proxies.
+        // TODO(gprusak): there are more corner cases to cover, because
+        // tier1_connect may also spawn TIER1 connections conflicting with
+        // tier1_advertise_proxies. It would be better to be able to await
+        // handshake on connection attempts, even if another call spawned them.
+        let _lock = self.tier1_advertise_proxies_mutex.lock().await;
         let accounts_data = self.accounts_data.load();
-        let Some(vc) = self.tier1_validator_config(&accounts_data) else {
-            return vec![];
+        let tier1 = self.tier1.load();
+        let vc = match self.tier1_validator_config(&accounts_data) {
+            Some(it) => it,
+            None => {
+                tracing::info!(target:"network","This IS NOT a TIER1 node");
```

Comment:
I'd add more info (that the node is neither a validator or producer, right?)

BTW - is it going to be spammy?

Reply:
I used this thing for debugging. Removed.